pkg/core/localize/locales/en/cmd/context.en.toml

```diff
@@ -0,0 +1,211 @@
+[context.cmd.shortDescription]
+one='Group, share and manage your rhoas services'
+
+[context.cmd.longDescription]
+one='''
+rhoas context commands allow developers to:
+
+  * Group services into contexts that can be used with a number of CLI commands.
+  * Manage different service contexts by switching, listing and removing service contexts 
+  * Share context with others to use the same set of services
+  * Generating configuration for connecting to the services from various platforms and tools
+
+The loading order of contexts follows these rules:
+
+  1. If $RHOAS_CONTEXT environment variable is set, then contexts are managed using only that file
+  2. Otherwise, "UserConfigDir/rhoas/contexts.json" is used for contexts
```

Comment:
`UserConfigDir` can point to different paths depending on env. Keeping it dynamic will cause problems for docs.

Reply:
`UserConfigDir` would not be that easy to transform to actual folder. 
I think we might need to print out location of the context file on status and document it here:

to view context location execute status command etc.